keyboards/duck/orion/v2_5/indicator_leds.h

```diff
@@ -0,0 +1 @@
+void indicator_leds_set(bool leds[8]);
```

Comment:
Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

Also, 
```suggestion
#pragma once

void indicator_leds_set(bool leds[8]);
```